### PR TITLE
add docs for slack threads options

### PR DESCRIPTION
**Pull request by ykamo001**
## DESCRIBE YOUR PR
We added the ability to toggle threads on or off at the integration level for all users! Adding documentation so that users know how to change the toggle, and we can use the link to help support users who ask.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/organization/integrations/index.mdx [hunk 1]
```diff
@@ -127,3 +127,11 @@ For more details, see the [full Integration Platform documentation](/organizatio
...
 | [AWS Lambda - Node (Automatic)](/organization/integrations/cloud-monitoring/aws-lambda/)   | X               | X                          |
 | [AWS Lambda - Node (Manual)](/platforms/javascript/guides/aws-lambda/)                      | X               | X                          |
+
+
+## Toggleable Features
+Some integrations have features that can be enabled or disabled through a toggle at the integration level. The toggle happens instantly across the feature, and defaults are dependent on the feature, per integration.
+
+Not every integration has toggleable features, but if they do, you can find them in the `Features` tab for the specific integration's settings.
+
+./img/integration-features-tab.png
```
Comment (cathteng) [inline docs/organization/integrations/index.mdx:137]:
```suggestion
![Integration features tab](./img/integration-features-tab.png)
```

File: docs/organization/integrations/notification-incidents/threads/index.mdx [hunk 2]
```diff
@@ -39,3 +39,11 @@ Threads will notify you when a metric alert has been triggered, and again when i
 Threads will keep your issue statuses in sync. The Threads notification will update when your issue is archived or resolved:
 
 ![Issue statuses synced](./img/threads-issue-alert.png)
+
+## Toggleable Features
+
+Threads can be enabled or disabled through the features tab in the Slack integration settings. Threads can be toggled for both Issue Alerts and Metric Alerts.
+
+./img/slack-threads-features.png
```
Comment (cathteng) [inline docs/organization/integrations/notification-incidents/threads/index.mdx:47]:
```suggestion
![Slack threads feature](./img/slack-threads-features.png)
```

File: docs/organization/integrations/index.mdx (hunk 1 above)
Comment (cathteng) [reply docs/organization/integrations/index.mdx:137]: this is how we get images to show up

File: docs/organization/integrations/index.mdx [hunk 3]
```diff
@@ -127,3 +127,11 @@ For more details, see the [full Integration Platform documentation](/organizatio
 | [AWS Lambda - Python (Manual)](/platforms/python/integrations/aws-lambda/)            | X               | X                          |
 | [AWS Lambda - Node (Automatic)](/organization/integrations/cloud-monitoring/aws-lambda/)   | X               | X                          |
 | [AWS Lambda - Node (Manual)](/platforms/javascript/guides/aws-lambda/)                      | X               | X                          |
+
+
+## Toggleable Features
+Some integrations have features that can be enabled or disabled through a toggle at the integration level. The toggle happens instantly across the feature, and defaults are dependent on the feature, per integration.
```
Comment (vivianyentran) [inline docs/organization/integrations/index.mdx:133]:
```suggestion
Some integrations have features that can be enabled or disabled through a toggle at the integration level. Toggling features on/off takes effect instantly across the integration. Defaults are dependent on the feature, per integration.
```
Not sure what you mean by "the toggle happens instantly." Not sure if you meant the above or maybe you mean "The toggle appears automatically for available features for each integration"? but that's also kinda assumed I think.

File: docs/organization/integrations/notification-incidents/threads/index.mdx [hunk 4]
```diff
@@ -39,3 +39,11 @@ Threads will notify you when a metric alert has been triggered, and again when i
...
 
 ![Issue statuses synced](./img/threads-issue-alert.png)
+
+## Toggleable Features
+
+Threads can be enabled or disabled through the features tab in the Slack integration settings. Threads can be toggled for both Issue Alerts and Metric Alerts.
+
+![Slack threads feature](./img/slack-threads-features.png)
+
+By default, the feature comes as enabled for both options.
```
Comment (vivianyentran) [inline docs/organization/integrations/notification-incidents/threads/index.mdx:49]:
```suggestion
By default, both features are enabled.
```

**Review by vivianyentran — APPROVED** [review]

File: docs/organization/integrations/index.mdx (hunk 3 above)
Comment (ykamo001) [reply docs/organization/integrations/index.mdx:133]: Yeah here I meant it takes effect instantly instead of there being a delay.

Comment (ykamo001) [reply docs/organization/integrations/index.mdx]: Used your suggestion, thank you!

File: docs/organization/integrations/notification-incidents/threads/index.mdx (hunk 4 above)
Comment (ykamo001) [reply docs/organization/integrations/notification-incidents/threads/index.mdx]: Used your suggestion, thank you!

File: docs/organization/integrations/notification-incidents/threads/img/slack-theards-features.png
Comment (AbhiPrasad) [inline docs/organization/integrations/notification-incidents/threads/img/slack-theards-features.png:1]: I think this needs to be named `docs/organization/integrations/notification-incidents/threads/img/slack-threads-features.png`

**Review by AbhiPrasad — CHANGES_REQUESTED** [review]

**Review by AbhiPrasad — APPROVED** [review]

File: docs/organization/integrations/notification-incidents/threads/img/slack-theards-features.png
Comment (ykamo001) [reply docs/organization/integrations/notification-incidents/threads/img/slack-theards-features.png:1]: good catch

**Pull request merged**